### DISCUSSION TITLE
iTunes Update Not working

### DISCUSSION THREAD

**Discussion by HoopsANZ** [dlrudie/Snip#531] [General]
Just updated to the app version of iTunes (thanks apple) and now snip isn't working. do I have to go back to Reulgar version 

Current installed version is 12130.9.2003.0 also will snip support the New Apple Music when it launches full version

**Comment by BioFUSION3k** [comment]
Hrmm, any idea what I would have to change to make this work with Apple's "Apple Music Preview" app on windoze? Would it be as simple as changing "ItunesApplication" to like "AppleMusicWinApplication" which seems to be the name in the apps folder...
